esp32/modules/easydraw.py

```diff
@@ -18,12 +18,10 @@ def msg(message, clear=False):
     if clear:
         ugfx.clear(ugfx.WHITE)
         ugfx.string(0, 0, message, "PermanentMarker22", ugfx.BLACK)
-        ugfx.set_lut(ugfx.LUT_FASTER)
-        ugfx.flush()
```

Comment:
That is correct, this makes sure that the title isn't redrawn when the message first appears on screen.

It does however require every app that uses this function to call it twice:

msg("title", True)
msg("message") -> this call redraws the screen